### ISSUE TITLE
the MAP of verification suddenly becomes 0 during training

### ISSUE THREAD

**Issue by yrn1126** [ultralytics/yolov5#8338]
![微信截图_20220625184517](https://user-images.githubusercontent.com/102804183/175770236-395cdd3b-43e7-463e-839b-4d518a18d94e.png)
![微信截图_20220625184551](https://user-images.githubusercontent.com/102804183/175770249-53807842-24aa-45a8-a662-30e045a0ba52.png)

 During the training of my own dataset, everything was normal. But at the 80th epoch, the MAP of the verification  became 0 suddenly, but the loss was still declining . After 10 epochs, it becomes normal again, but the MAP was much lower than before. I feel confused about this question.

**Comment by glenn-jocher** [comment]
@yrn1126 training instabilities, your losses are increasing. Probably due to incorrect labels or modified hyperparameters.
